src/renderers/server-render.jsx

```diff
@@ -81,43 +81,33 @@ const getRouterRenderer = (
 	let statusCode;
 	const context = {};
 
-	try {
-		appMarkup = ReactDOMServer.renderToString(
-			<StaticRouter
-				basename={baseUrl}
-				location={location}
-				context={context}
-			>
-				<PlatformApp store={store} routes={routes} />
-			</StaticRouter>
-		);
-
-		if (context.url) {
-			// redirect
-		}
-
-		// all the data for the full `<html>` element has been initialized by the app
-		// so go ahead and assemble the full response body
-		result = getHtml(
-			baseUrl,
-			assetPublicPath,
-			clientFilename,
-			initialState,
-			appMarkup
-		);
-
-		statusCode = NotFound.rewind() ||  // if NotFound is mounted, return 404
-			200;
-
-	} catch(error) {
-		// log the error stack here in dev to make it a little more legible
-		// - prod will get stackdriver formatting
-		if (process.env.NODE_ENV !== 'production') {
-			console.error(error.stack);
-		}
-		throw error;
+	appMarkup = ReactDOMServer.renderToString(
```

Comment:
the previous `catch` was just a way to log the error in a readable format and then re-throw it, so now that's unnecessary